template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/schemaregistry/SchemaRegistryServiceConfigProvider.java

```diff
@@ -33,24 +33,29 @@ public class SchemaRegistryServiceConfigProvider extends AbstractRdsRoleConfigPr
 
     static final String DATABASE_PASSWORD = "database_password";
 
+    static final String DATABASE_JDBC_URL_OVERRIDE = "database_jdbc_url_override";
+
     static final String RANGER_PLUGIN_SR_SERVICE_NAME = "ranger.plugin.schema-registry.service.name";
 
     @Override
     public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor templateProcessor, TemplatePreparationObject source) {
         String cdhVersion = source.getBlueprintView().getProcessor().getStackVersion() == null ?
                 "" : source.getBlueprintView().getProcessor().getStackVersion();
+        List<ApiClusterTemplateConfig> config = new ArrayList<>();
         if (isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_2_0)) {
             RdsView schemaRegistryRdsView = getRdsView(source);
-            return List.of(
-                    config(DATABASE_TYPE, dataBaseTypeForCM(schemaRegistryRdsView.getDatabaseVendor())),
-                    config(DATABASE_NAME, schemaRegistryRdsView.getDatabaseName()),
-                    config(DATABASE_HOST, schemaRegistryRdsView.getHost()),
-                    config(DATABASE_PORT, schemaRegistryRdsView.getPort()),
-                    config(DATABASE_USER, schemaRegistryRdsView.getConnectionUserName()),
-                    config(DATABASE_PASSWORD, schemaRegistryRdsView.getConnectionPassword())
-            );
+            config.add(config(DATABASE_TYPE, dataBaseTypeForCM(schemaRegistryRdsView.getDatabaseVendor())));
```

Comment:
Not a big deal but it would perhaps be sufficient to set `DATABASE_TYPE` in the `else` branch, i.e. when SSL is not enabled.

Reply:
required